models/Drone.model.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const { Schema, model } = mongoose;
+const droneSchema = new Schema(
+  {
+    name: String,
+    propellers: Number,
+    maxSpeed: Number
+  },
+  {
+    timestamps: true
+  }
+);
+
+const Drone = mongoose.model('Drone', droneSchema);
```

Comment:
This is correct, I just want to mention that since you exported function model on line 2 then you probably could have referred to it without using mongoose.
```
const Drone = model('Drone', droneSchema);